types/node-red/index.d.ts

```diff
@@ -1,221 +1,231 @@
-// Type definitions for node-red 0.20
-// Project: http://nodered.org
-// Definitions by: Anders E. Andersen <https://github.com/andersea>
-//                 Thomas B. Mørch <https://github.com/tbowmo>
-//                 Bernardo Belchior <https://github.com/bernardobelchior>
```

Comment:
can we add those owners back?

Reply:
Makes sense as the previous version will be kept supported. Done.